src/server/conn.rs

```diff
@@ -271,7 +277,9 @@ impl Conn {
         where T: RaftStoreRouter,
               S: StoreAddrResolver
     {
+        let n = self.send_buffer.len();
         msg.encode_to(&mut self.send_buffer).unwrap();
+        CONN_PENDING_SEND_BYTES_COUNTER.inc_by((self.send_buffer.len() - n) as f64).unwrap();
```

Comment:
No, here we will calculate the rate of pending bytes in Prometheus, it will be increased all time.